paloesg/excide
# Display errors accessing to batch page due to missing templates

# Description
Problem: User cannot access Batch INDEX page because total_actions of workflows is missing. The problem comes from the fact that there were no workflows in batch (which could be a concern in the future), which leads to the progress bar counting nil value. Therefore, one batch with no workflows would cause the user to not access the rest of the batch

Solution: Add condition that if batches doesn't have workflow, the progress bar would display "ERROR ~ please delete batch" message. From the backend, check for workflows in the batch

Trello link: https://trello.com/c/{card-id}

## Remarks
- The concern is because the batch is created differently from the workflow, thus wondering if there would be a possibility that the workflow created is not link to the batch

# Testing
- Create batch and then remove the workflows to reproduce the error, and from there try to prevent the error
- Create multiple batch and check progress bar accuracy

## Checklist:
